orca_test/orca_test.py

```diff
@@ -396,6 +399,33 @@ def assert_column_is_primary_key(table_name, column_name):
     return
 
 
+def assert_column_is_unique(table_name, column_name):
+    """
+    Assert that column's values are unique. 
+    
+    Parameters
+    ----------
+    table_name : str
+    column_name : str
+    
+    Returns
+    -------
+    None
+    
+    """
+    assert_column_can_be_generated(table_name, column_name)
+    
+    ds = get_column_or_index(table_name, column_name)
+    
+    print(ds)
```

Comment:
Was the print() statement for testing? Might want to remove it for general usage